scripts/gulpfiles/config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Common configuration for Gulp scripts.
+ */
+
+var path = require('path');
+
+// Paths are all relative to the repository root.  Do not include
+// trailing slash.
+module.exports = {
+  // Directory to write compiled output to.
+  BUILD_DIR: 'built',
```

Comment:
Why not use "build" here? I thought using build was a common pattern, but it could just be that we happen to be using it in Blockly Samples.

Reply:
I don't have a strong opinion about this.  I picked "built" over "build" (after also considering "bin", "out", "compiled", etc.) because "build" sounded like it might contain build _tools_, whereas "built" seemed like output.  I wasn't able to find any info about what conventions are typically used here—but if we're using "build" in other packages for ephemeral compiler output then that's great and I'll go with that.

I was going to write output directly to `dist/`, but the files in `built/msg/` differ from the ones in `builld/msg/js/` (and `msg/js/`) for reasons I am not yet  clear on, so I've held off on that for now.

Reply:
I've renamed this to "build", since it seems that is more commonly used.